tests/data/bundle.yaml

```diff
@@ -7,7 +7,7 @@ machines:
   '1':
     constraints: cores=4 mem=4G root-disk=16G
     series: {{ series }}
-applications:
+services:
```

Comment:
`services` in bundles is deprecated. This should stay as `applications`.

Reply:
i think i saw this in "charmed-kubernetes-master" (https://github.com/charmed-kubernetes/charm-kubernetes-master/blob/master/tests/data/bundle.yaml#L10).  Will revert as `applications` is preferred